papers/source/d1664.bs

```diff
@@ -243,7 +257,22 @@ Append to §17.3.1 General [[support.limits.general](http://eel.is/c++draft/supp
 Insert into §24.4.4 Ranges [[range.range](http://eel.is/c++draft/range.req#range.range)]'s after clause 7, one additional clause:

```

Comment:
Note that on http://eel.is/c++draft/range.req#range.range 24.4.4. doesn't have clause 7, did you mean 24.4.2? I am actually unsure of were best to put it.

  * It would fit below `forwarding-range` in 24.4.2
  * It would fit with the views in 24.4.4 if we change the requirements to require `View<R>` above
  * It would fit below the rest in 24.4.5

Reply:
Yes, I mean 24.4.2.